azure-pipelines.yml

```diff
@@ -0,0 +1,68 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - main
+      - release/*
+
+resources:
+  repositories:
+    - repository: niveristand-custom-device-build-tools
+      type:       github
+      ref:        main
+      endpoint:   nivs-custom-devices
+      name:       ni/niveristand-custom-device-build-tools
+
+stages:
+  - template: azure-templates/stages.yml@niveristand-custom-device-build-tools
+    parameters:
+      submodules: true
```

Comment:
This won't work until https://github.com/ni/niveristand-custom-device-build-tools/pull/162 is submitted.

Reply:
merged